### PR TITLE
Review

### PR DESCRIPTION
**Pull request by parshant786**

### CODE REVIEW

**Comment by parshant786** [comment]
up to date dev

**Comment by parshant786** [comment]
upto date dev branch

File: index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const app = express();
+//do something as
+
+app.use(express.static("public"));
+
+app.listen(3009, () => {
+  console.log("listening....");
```
Comment (MohitJainStanza) [inline index.js:8]: keep port in a variable and also keep it in console also

File: package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.18.1",
+    "nodemon": "^2.0.16"
```
Comment (MohitJainStanza) [inline package.json:13]: remote `^` from here

**Pull request merged**